notebooks/220-license-plate-recognition/220-license-plate-recognition.ipynb

```diff
@@ -0,0 +1,555 @@
+    "\n",
+    "License plate recognition model helps you get the chinese license plate number precisely in no time. The input of the color license plate image can be any size. It will be resized and augmented before put into the model. After matching the result to correct character, we can get the license plate number. In the notebook we show how to create the following pipeline:\n",
+    "\n",
+    "![flowchart.png](attachment:91de97db-4ae9-4fae-8c72-a530cfedcb5f.png)\n",
+    "\n",
+    "> Note: Augmentation method on image is optional and it may lead to the wrong recognition result. Thus, we recommend you to use it under special conditions like image overexposure or too dark.\n",
+    "\n",
+    "Example image data comes from [CCPD (Chinese City Parking Dataset, ECCV)](https://github.com/detectRecog/CCPD)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import collections\n",
+    "import os\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from IPython import display\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Model\n",
+    "\n",
+    "### Download the Model\n",
+    "\n",
+    "We use `omz_downloader`, the tool from the `openvino-dev` package to download the selected model. It will be retrieved from the cache if the model is already downloaded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "\n",
+    "# model name as named in Open Model Zoo\n",
+    "model_name = \"license-plate-recognition-barrier-0001\"\n",
+    "\n",
+    "# it will be retrieved from the cache if the model is already downloaded.\n",
+    "download_command = (\n",
+    "    f\"omz_downloader \"\n",
+    "    f\"--name {model_name} \"\n",
+    "    f\"--output_dir {base_model_dir} \"\n",
+    "    f\"--cache_dir {base_model_dir}\"\n",
+    ")\n",
+    "\n",
+    "# this code is provided for the first download of the model\n",
+    "#! $download_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Convert the Model\n",
```

Comment:
@zRAINj the models are downloaded in OpenVINO IR format. This step is not needed